phoenix-queryserver/pom.xml

```diff
@@ -147,6 +147,10 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.curator</groupId>
+      <artifactId>curator-client</artifactId>
```

Comment:
Assuming you follow my suggestion about using service loader, the curator dependencies can be removed from this pom.xml.